examples/tutorials/02_pipeline_operations.ipynb

```diff
@@ -14,18 +14,17 @@
    "outputs": [],
```

Comment:
<p>why do you always use subplot_kw with empty lists? May be you can use them as a default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>